### ISSUE TITLE
WordPressCli should be updated to use new APIs

### ISSUE THREAD

**Issue by jhavlin** [junichi11/netbeans-wordpress-plugin#48]
This exception has been reported to NetBeans Bugzilla:

```
java.lang.NoSuchMethodError: org.netbeans.modules.php.api.executable.PhpExecutable.run(Lorg/netbeans/api/extexecution/ExecutionDescriptor;Lorg/netbeans/api/extexecution/ExecutionDescriptor$InputProcessorFactory;)Ljava/util/concurrent/Future;
   at org.netbeans.modules.php.wordpress.commands.WordPressCli.getCommands(WordPressCli.java:383)
   at org.netbeans.modules.php.wordpress.commands.WordPressCli.updateCommands(WordPressCli.java:363)
   at org.netbeans.modules.php.wordpress.commands.WordPressCli.getCommands(WordPressCli.java:356)
   at org.netbeans.modules.php.wordpress.WordPressModuleInstall$1.run(WordPressModuleInstall.java:121)
   at org.openide.util.RequestProcessor$Task.run(RequestProcessor.java:1443)
   at org.netbeans.modules.openide.util.GlobalLookup.execute(GlobalLookup.java:68)
```

See https://netbeans.org/bugzilla/show_bug.cgi?id=254924.

The WordPress plugin was installed in NetBeans 8.1 Beta. API of `org.netbeans.modules.php.api.executable.PhpExecutable.run()` has changed and is not compatible with the plugin (`InputProcessorFactory2` must be now used instead of `InputProcessorFactory`).

Please check recent API changes and update the plugin.
Thank you.

**Comment by junichi11** [comment]
Thanks for letting me know!

I've already fixed that : #43 (https://github.com/junichi11/netbeans-wordpress-plugin/commit/f9f56ef268ffd436ce3d17a09c68b81473ddc740#diff-0eb09360106e8022e691cc566149fdc2R538)
But I have not released a new version for NB81 yet. So, I'll upload it to Plugin Portal later.

Thanks!

**Comment by tmysik** [comment]
@junichi11 @jhavlin 
Thanks gyus!
